source/clear-linux/concepts/security.rst

```diff
@@ -0,0 +1,205 @@
+.. _security:
+
+Clear Linux\* Security 
+**********************
+
+Clear Linux\*, as it is distributed and maintained upstream, aims to make 
```

Comment:
Please remove the phrase, "as it is distributed [...] upstream,". It's already understood. It's documented elsewhere.